msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
```

Comment:
Fair enough but we use gitsetup.exe at my company. This effort will be a massive waste of time for me if we lose functionality. I'll have to go back and still maintain gitsetup.exe. My hope was in the end we'd get the best of all worlds in one place.

This is important to me.